galpy/df/__init__.py

```diff
@@ -66,4 +66,7 @@
 osipkovmerrittdf = osipkovmerrittdf.osipkovmerrittdf
 osipkovmerrittNFWdf = osipkovmerrittNFWdf.osipkovmerrittNFWdf
 constantbetadf = constantbetadf.constantbetadf
+basestreamspraydf = streamspraydf.basestreamspraydf
```

Comment:
This is a bit tricky. I did this because I want to test the `basestreamspraydf.spray_df()` method. Otherwise, that part of codes would be reported as uncovered by tests.

Normally, I would decorate the top-level class as an abstract class (and make the `spray_df()` method an abstract method) so that I don't need to bother the test coverage. But this is not how `galpy` works. Do have any suggestions?

Reply:
Thanks for the suggestions! 

For 1: I have redirected `galpy.df.streamspraydf` to `fardal15spraydf` for backwards compatibility, so this won't work unless I change the module's name

3 is a good idea! I didn't know it would be ignored. But good to know!

Reply:
Not sure why solution 3 is not working. I will use `# pragma: no cover`. But this is a small edit. I'll save this for the next commit to save test times.

Reply:
It's because you have to do `raise NotImplementedError` rather than raise a warning.